src/doc/en/reference/references/index.rst

```diff
@@ -2921,6 +2921,9 @@ REFERENCES:
                  "Pyjamask"
                  https://csrc.nist.gov/CSRC/media/Projects/Lightweight-Cryptography/documents/round-1/spec-doc/Pyjamask-spec.pdf
 
+.. [Gouvea] Fernando Q. Gouvêa, *$p$-adic Numbers: An Introduction*,
```

Comment:
```suggestion
.. [Gou2020] Fernando Q. Gouvêa, *$p$-adic Numbers: An Introduction*,
```

Reply:
First three letters followed by the year is what we usually do for single-authored works